settings.gradle

```diff
@@ -32,10 +32,11 @@ include 'ethereum:referencetests'
 include 'ethereum:rlp'
 include 'ethereum:trie'
 include 'metrics'
-include 'orion'
 include 'pantheon'
 include 'services:kvstore'
 include 'services:queue'
 include 'services:util'
 include 'testutil'
 include 'util'
+include 'quickstart'
```

Comment:
what is quickstart that has been added here?
Also they were in alphabetical order, might be helpful to maintain this ordering.

Reply:
Removed the quickstart and sorted: https://github.com/PegaSysEng/pantheon/pull/745/commits/775fdfab7320b2652ea40bfd4555dda6848a4baa